pkg/rtc/mediatrack.go

```diff
@@ -208,10 +208,6 @@ func (t *MediaTrack) ToProto() *livekit.TrackInfo {
 	return t.MediaTrackReceiver.TrackInfoClone()
 }
 
-func (t *MediaTrack) UpdateCodecCid(codecs []*livekit.SimulcastCodec) {
```

Comment:
Just removed the forwarding as `MediaTrackReceiver` is embedded in `MediaTrack`.